doc/cheatsheet/dp_bitmask.md

```
@@ -325,7 +535,8 @@ public boolean canPartitionKSubsets(int[] nums, int k) {
 | **Assignment** | dp[mask] = cost to assign tasks in mask | Assign next task to worker | LC 1723, LC 1986 |
 | **Subset Selection** | dp[mask] = ways/cost for subset mask | Include/exclude next item | LC 691, LC 1434 |
 | **Partition** | dp[mask] = can partition mask into groups | Form complete groups | LC 698, LC 1681 |
-| **Profile DP** | dp[i][mask] = state at row i with column mask | Process row by row | Tiling problems |
+| **Set Cover** | dp[cover] = cheapest set of items reaching that cover | Union in one more item | LC 1125, LC 691 |
+| **Profile DP** | dp[i][mask] = state at row i with column mask | Process row by row | LC 1349, tiling problems |
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "mask"

(MD052, reference-links-images)